src/components/LenderProfile/LenderSummary.vue

```diff
@@ -53,20 +60,97 @@
 				</div>
 			</div>
 		</div>
+		<kv-lightbox
+			:visible="lightboxVisible"
+			@lightbox-closed="lightboxClosed"
+			:title="lightboxTitle"
+		>
+			<!-- eslint-disable max-len vue/singleline-html-element-content-newline -->
+			<template v-if="!isLoggedIn">
+				<p>
+					You must be signed in and have made loans on Kiva first to send a Lender Message. This helps us reduce the amount of SPAM and unwanted messages in the system.
+				</p>
+			</template>
+
+			<template v-if="isLoggedIn && loansNumber == 0">
```

Comment:
I think we'll need this to be `v-else-if` for the Vue 3 migration. Same with below on line 86.